client/src/components/CreatePost.tsx

```diff
@@ -8,22 +8,39 @@ import TextField from "@mui/material/TextField";
 import Divider from "@mui/material/Divider";
 import Typography from "@mui/material/Typography";
 import Tooltip from "@mui/material/Tooltip";
-
+import Dialog from "@mui/material/Dialog";
+import { AppBar, IconButton } from "@mui/material";
+import { ArrowBack } from "@mui/icons-material";
 import PreviewPopUp from "./PreviewPopUp";
 
+import { api_v1 } from "../api/v1";
+
 function CreatePost() {
   // create hooks for input fields
   const [form, setForm] = useState({
     title: "",
     body: "",
     file: "",
     tags: "",
-    eventCapacity: "",
-    eventLocation: "",
+    capacity: "",
+    location: "",
   });
 
   const [openPopup, setOpenPopup] = useState(false);
 
+  const handleSubmit = () => {
+    api_v1
+      .post("/posts/", form)
+      .then((res) => {
+        console.dir(res);
```

Comment:
I don't think it's a good idea to log an entire res object to the console where its plainly visible to anyone who opens a dev console. I think you just need to handle the data in res, you don't necessarily need to log it to the console.

Reply:
Agreed. But this should change with the clientApi class either way.